tasks/internal_reconcile_report_orphan/src/adapters/api/aws.py

```diff
@@ -0,0 +1,166 @@
+import json
+import os
+from http import HTTPStatus
+from typing import Dict, Union, Any, List
+
+import fastjsonschema as fastjsonschema
+import orca_shared
+from cumulus_logger import CumulusLogger
+from fastjsonschema import JsonSchemaException
+from orca_shared.database.adapters.api import get_configuration
+from orca_shared.database.use_cases import create_user_uri
```

Comment:
Should this be `from orca_shared.database.use_cases.create_postgres_connection_uri import create_user_uri` ?